resources/launchscripts/osmc/launch.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/sh
+
+resume() {
+  killall -CONT kodi.bin
+}
+
+trap resume EXIT INT TERM
+
+killall -STOP kodi.bin
+
+mv logfile logfile.old
+LD_PRELOAD="/usr/local/lib/libmoonlight-aml.so /usr/osmc/lib/libamavutils.so /usr/osmc/lib/libamadec.so /usr/osmc/lib/libamcodec.so" \
+  moonlight stream \
+  -platform aml \
+  -app "$@" >> logfile 2>&1
```

Comment:
I based this on https://github.com/moonlight-stream/moonlight-embedded/issues/573#issuecomment-2196759628

If I understand correctly, `-platform aml` forces the video to run with the Amlogic hardware
.